src/rgw/rgw_op.cc

```diff
@@ -2681,6 +2681,35 @@ void RGWDeleteBucket::execute()
     }
   }
 
+  vector<rgw_bucket_dir_entry> objs;
+  string prefix = "", marker = "", delimiter = "";
+  map<string, bool> common_prefixes;
+  bool is_truncated;
+  static MultipartMetaFilter mp_filter;
+  int max_uploads = 1000;
```

Comment:
The idea here is to just return ENOTEMPTY, even if a single incomplete upload is found. The `max_uploads` number here is just to list objects, and not to delete them.